src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.jsx

```diff
@@ -93,17 +93,26 @@ const SelectHareDropDown = ({
       ],
   );
 
-  const getFinalPopulationSize = (selectedHareItem) => {
-    const hareBreakdownPart1 = hareResults[0].data.concept_breakdown.find(
-      (hare) => selectedHareItem.value === hare.concept_value,
-    ).persons_in_cohort_with_value;
+  const getFinalPopulationSizes = (selectedHareItem) => {
     if (outcome.variable_type === 'concept') {
-      return hareBreakdownPart1;
+      // Quantitative outcome scenario:
+      const finalCohortSize = hareResults[0].data.concept_breakdown.find(
+        (hare) => selectedHareItem.value === hare.concept_value,
+      ).persons_in_cohort_with_value;
+      return [{ population: 'Total', size: finalCohortSize }];
     }
-    const hareBreakdownPart2 = hareResults[1].data.concept_breakdown.find(
+    // Dichotomous (aka Case/Control) outcome scenario:
+    const finalControlCohortSize = hareResults[0].data.concept_breakdown.find(
+      (hare) => selectedHareItem.value === hare.concept_value,
+    ).persons_in_cohort_with_value;
+    const finalCaseCohortSize = hareResults[1].data.concept_breakdown.find(
       (hare) => selectedHareItem.value === hare.concept_value,
     ).persons_in_cohort_with_value;
-    return hareBreakdownPart1 + hareBreakdownPart2;
+    return [
+      { population: 'Control', size: finalControlCohortSize },
+      { population: 'Case', size: finalCaseCohortSize },
+      { population: 'Total', size: finalControlCohortSize + finalCaseCohortSize },
+    ];
   };
 
   const onChange = (h) => {
```

Comment:
**Question** On line 121 we're hard coding the concept_id passed to the dispatch with action UPDATE_SELECTED_HARE. Is this correct?

Reply:
good catch. Fixed.

Reply:
good catch.